### ISSUE TITLE
How to change the $ symbol in `TYPE_CURRENCY`?

### ISSUE THREAD

**Issue by Gomesz785** [fgsl/eyedatagrid#4]
What should I modify?

**Comment by fgsl** [comment]
@Gomesz785, excuse me, but I didn't find TYPE_CURRENCY in the source code. Could you show a sample?

**Comment by Gomesz785** [comment]
> @Gomesz785, excuse me, but I didn't find TYPE_CURRENCY in the source code. Could you show a sample?

I am currently in work, I believe it was some documentation. I will check the issue file today and reply.

**Comment by Gomesz785** [comment]
Sorry, it was something like
>`$TYPE_CURRENCY`

or
>`$TYPE_DOLLAR`

or
>**TYPE_CURRENCY**
>
>```
>$somethingTYPE_CURRENCYsomething
>```
or
>**TYPE_DOLLAR**
>
>```
>$somethingTYPE_CURRENCYsomething
>```

I had solved the issue by using a other data format, I used a proprietary work script for doing that. I won't be able point the exact issue because it is deleted in my issue log (above part was just a memory), hence help you to solve that.

I see that this can be useful for other people, so thank you.
